docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -126,8 +126,8 @@ unit time.
         is determined in the
         intrapartum step of the :ref:`pregnancy model
         <other_models_pregnancy_closed_cohort_mncnh>`)
-    * - neonate survived
-      - The child simulant survived for the first 28 days of life
+    * - neonate did not die of preterm
+      - The child simulant did not die of preterm birth complications during the first 28 days of life
```

Comment:
Don't we need to split this into 0-7 days and 7-28 days just like as described on the ACMR page?

Reply:
as above, I will leave this for now